### PR TITLE
Add Concepts: Keywords

### LINKED ISSUE #4679: [Concept Entry] MySQL keywords
### Reason/inspiration (optional)

We would like a new entry on the `Keywords` concept under MySQL. The entry should go in a new file under `docs/content/mysql/concepts/keywords/keywords.md`.

The entry should include:

- An introductory paragraph that describes the concept
- A ‘Syntax’ section that includes the syntax for the concept (if applicable)
- An ‘Example’ section that shows the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Danielmnm1**
### Description

- Added Concepts
- Added Keywords
- Added keywords.md describing the most important keywords in mysql and adding examples.

### Issue Solved

Closes #4679  

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/mysql/concepts/keywords/keywords.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# MySQL Keywords
+
+MySQL **keywords** are predefined words that have special meaning in the SQL language. They are used to perform various operations within the database, such as creating tables, inserting data, querying the database, updating records, and more. Examples of MySQL keywords include SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, and ALTER.
+
+## Importance and Usage
+
+Keywords define the structure and syntax of SQL queries. They help organize the commands in a way that the MySQL database can interpret and execute. Each keyword performs a specific function such as selecting data (`SELECT`), inserting new records (`INSERT`), or modifying existing records (`UPDATE`).
+Using keywords ensures that your SQL queries are standarized, this enhances readability and maintainability of your code. 
+
+### Example
```
Comment (letsbuildanything) [inline content/mysql/concepts/keywords/keywords.md:24]: @Danielmnm1, 
Please add two separate sections: ##Syntax and ##Example.
In the Example section, include a table with a small number of rows and columns. like:

``` sql
CREATE TABLE products (
    name VARCHAR(50),
    price DECIMAL(10, 2),
    quantity_in_stock INT
);

INSERT INTO products (name, price, quantity_in_stock) VALUES
('Product A', 19.99, 10),
('Product B', 5.99, 20),
('Product C', 49.99, 5);
```

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@Danielmnm1 
I've requested some changes. Please take a moment to address them. If you have any concerns or reasons to reject specific suggestions, feel free to discuss them.
Also, Please refer to [style-guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md). It is a general guide to entry standards, which will help reduce the number of potential edits that will further need to be made to your PR.

**Comment by Danielmnm1** [comment]
I fixed the branch issue, please let me know your comments. Thank you.

**Comment by ishg-153** [comment]
@letsbuildanything Please check if your required changes are reflected.

File: content/mysql/concepts/keywords/keywords.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+SET salary = 80000.00
+WHERE id = 1;
+```
+
+### DELETE
+
+```SQL
+DELETE FROM employees
+WHERE id = 1;
+```
```
Comment (letsbuildanything) [inline content/mysql/concepts/keywords/keywords.md:111]: I suggest combining the use cases of different keywords into one comprehensive example. Like this:

> ## Example
> 
> The following example demonstrates the use of several MySQL keywords:
> 
> ```SQL
> -- Create a new table
> CREATE TABLE employees (
>     id INT AUTO_INCREMENT PRIMARY KEY,
>     first_name VARCHAR(50),
>     last_name VARCHAR(50),
>     position VARCHAR(100),
>     salary DECIMAL(10, 2),
>     hire_date DATE
> );
> 
> -- Insert a new employee
> INSERT INTO employees (first_name, last_name, position, salary, hire_date)
> VALUES ('John', 'Doe', 'Software Engineer', 75000.00, '2024-05-25');
> 
> -- Select specific employees
> SELECT first_name, last_name
> FROM employees
> WHERE position = 'Software Engineer';

```

- Feel free to edit and improve as needed.  
- Wherever possible, Please add the output section.

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@Danielmnm1 I've requested some more changes. Please take a moment to address them. If you have any concerns or reasons to reject specific suggestions, feel free to discuss them.

**Review by letsbuildanything — APPROVED** [review]
Hey @Danielmnm1,
Due to inactivity, I've proceeded by incorporating the requested changes.
It's ready for the second round of review.

**Review by Sriparno08 — APPROVED** [review]
Looks good, @Danielmnm1! 👍

**Pull request merged**